cicd/docupdate.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/env bash
```

Comment:
docupdate.sh was moved to on_nightly.sh. You can probably remove this file `git rm cicd/docupdate.sh`. and then add the relevant changes to https://github.com/ZOSOpenTools/meta/blob/main/cicd/on_nightly.sh#L16 and then commit it

Reply:
Ah my bad, I had forgotten about that